### PR TITLE
README: Add info about jivthesh

### PR DESCRIPTION
**Pull request by jivthesh**
i have changed my commits into one .

### CODE REVIEW

**Comment by tushar5526** [comment]
Looks good, wait for the mods to merge you !

File: jivthesh.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (tushar5526) [inline jivthesh.md:1]: Hey @jivtesh looks like you forgot to add your info, did you used `git add ` after the changes

Comment (jivthesh) [reply jivthesh.md:1]: yes

Comment (tushar5526) [reply jivthesh.md:1]: jitvesh.md is empty

**Pull request closed without merging**